scripts/layouts_tool/layout.h

```diff
@@ -0,0 +1,32 @@
+#ifndef LAYOUT_H
+#define LAYOUT_H
+
+#include <QString>
+#include <QX11Info>
+#include <QStringList>
+#include "defs.h"
+
+struct KbdKeyChar {
+    char16_t character;
+    bool is_dead;
+};
+
+typedef const KbdKeyChar (*KeyboardLayoutPointer)[2];
```

Comment:
Ah yes. I moved both the typedef and the KbdKeyChar struct declaration to `layouts.h` but forgot to actually remove it from its first place, thanks. It's fixed